package.json

```diff
@@ -104,7 +104,7 @@
 		"@wordpress/html-entities": "2.8.0",
 		"@wordpress/i18n": "3.15.0",
 		"@wordpress/is-shallow-equal": "1.8.0",
-		"@wordpress/scripts": "12.2.1",
+		"@wordpress/scripts": "^13.0.0",
```

Comment:
Needed to update this in order to get the latest `@wordpress/eslint-plugin` ruleset.